addon/services/user-activity.js

```diff
@@ -46,14 +45,13 @@ export default Service.extend(Evented, {
   init() {
     this._super(...arguments);
 
-    if (testing) { // Do not throttle in testing mode
-      this.set('EVENT_THROTTLE', 0);
-    }
-    this.setProperties({
-      _boundEventHandler: this.handleEvent.bind(this),
-      _eventsListened: A(),
-      _throttledEventHandlers: {}
-    });
+    // Do not throttle in testing mode
+    this._EVENT_THROTTLE = testing ? 0 : 100;
```

Comment:
I err'd on the wrong side of the law. I'll take it back to being an overridable property! The `CAPS` screamed `const` at me and I wasn't clear that it was public API. Good catch, thanks.